### DISCUSSION TITLE
InvalidIndexError

### DISCUSSION THREAD

**Discussion by anas031** [AammarTufail/machinelearning_ka_chilla#11] [Q&A]
I am trying to plot a basic line but getting this error-
InvalidIndexError: (slice(none, none, none),  none)
@1:19:58 in ML101 video
can anybody guide me here where am I going wrong?
![1](https://user-images.githubusercontent.com/100215810/159279097-0f37896d-1ee9-4f9b-a5ea-677f7f8dbb75.PNG)
![2](https://user-images.githubusercontent.com/100215810/159279105-d75d782c-43b9-4cd7-b6aa-941818a87446.PNG)

**Comment by Faizali100** [comment]
How did you initialize your input and output? try to solve this by calling
*.values()* function/method to the input/output of iloc . like, X =
dataset.iloc[:, :-1].values
